### DISCUSSION TITLE
Problem with Wallet Address to Firebase connection

### DISCUSSION THREAD

**Discussion by kylesdesigns** [Web3Auth/Web3Auth#1112] [Help]
Hi guys, 

We are integrating email web3auth in our typescript project with popup.  We used "email_passwordless" as provider.
But if we try to login with 2 different email accounts in google, we end up getting the same address.

If we login with "aaa@mail.co", the address is 0x0000
and if we log in with "bbb@mail.co", the address is also 0x0000.

We have tried to logout from web3auth but didn't fix the issue.

Has this happened to anyone else and if so how did you fix it?

**Comment by shahbaz17** [comment]
Hey @kylesdesigns 

Could you share the code snippet where the web3auth initialization and login are happening?
